cmd/cmd.go

```diff
@@ -31,6 +33,8 @@ const (
 	EnvDatabaseDSN     = "GITPODS_DATABASE_DSN"
 	EnvGRPCAddr        = "GITPODS_GRPC_ADDR"
 	EnvHTTPAddr        = "GITPODS_HTTP_ADDR"
+	EnvSSHAddr         = "GITPODS_SSH_ADDR"
```

Comment:
These should be in alphabetical order. However, all env vars are going to be removed and only the once really needed as env var option will be left.